userland/examples/lua-hello/Makefile

```diff
@@ -0,0 +1,20 @@
+# Makefile for user application
+
+# Specify this directory relative to the current application.
+TOCK_USERLAND_BASE_DIR = ../..
+
+# Which files to compile.
+C_SRCS := $(wildcard *.c)
+
+# External libraries used
+EXTERN_LIBS += $(TOCK_USERLAND_BASE_DIR)/lua53
```

Comment:
Would it be better to put this in a `lua` directory so we could update someday without imposing churn everywhere?